core/state_processor_test.go

```diff
@@ -485,8 +485,8 @@ func TestProcessVerkle(t *testing.T) {
 
 	txCost1 := params.TxGas
 	txCost2 := params.TxGas
-	contractCreationCost := intrinsicContractCreationGas + uint64(7700 /* creation */ +2939 /* execution costs */)
-	codeWithExtCodeCopyGas := intrinsicCodeWithExtCodeCopyGas + uint64(7000 /* creation */ +299744 /* execution costs */)
+	contractCreationCost := intrinsicContractCreationGas + uint64(5600+700+700+700 /* creation with value */ +2739 /* execution costs */)
+	codeWithExtCodeCopyGas := intrinsicCodeWithExtCodeCopyGas + uint64(5600+700 /* creation */ +302044 /* execution costs */)
```

Comment:
- The original `7000` was changed to `5600+700=6300` which makes sense since this tx doesn't send value. 
- The `299744` to `302044` is related to the fact that this `codeWithExtCodeCopyGas` does to internal `CREATE` calls which are affected by #393, so the change comes from there.

Unfortunately, all this is mixed up with the apparently wrong original execution cost of the `contractCreationCost` mentioned in the above comment.